teamcity: failed tests on master: test/TestBackupRestoreInterleaved, test/TestBackupRestoreInterleaved/all_tables_in_interleave_hierarchy

The following tests appear to have failed:

[#306083](https://teamcity.cockroachdb.com/viewLog.html?buildId=306083):

```
--- FAIL: test/TestBackupRestoreInterleaved (2.060s)

------- Stdout: -------
W170726 14:59:23.663603 37334 server/server.go:303  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170726 14:59:23.663787 37334 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170726 14:59:23.665931 37334 server/config.go:523  [n?] 1 storage engine initialized
I170726 14:59:23.665956 37334 server/config.go:525  [n?] RocksDB cache size: 512 MiB
I170726 14:59:23.665970 37334 server/config.go:525  [n?] store 0: in-memory, size 100 MiB
I170726 14:59:23.666199 37334 server/node.go:432  [n?] store [n0,s0] not bootstrapped
I170726 14:59:23.668325 37334 server/node.go:367  [n?] **** cluster 62b205ee-8db3-4ead-8cb3-6eb2bffa4361 has been created
I170726 14:59:23.668346 37334 server/node.go:368  [n?] **** add additional nodes by specifying --join=127.0.0.1:45878
I170726 14:59:23.669176 37334 storage/store.go:1262  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170726 14:59:23.669214 37334 server/node.go:445  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0 WritesPerSecond:0}
I170726 14:59:23.669285 37334 server/node.go:329  [n1] node ID 1 initialized
I170726 14:59:23.669325 37334 gossip/gossip.go:297  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45878" > attrs:<> locality:<> 
I170726 14:59:23.669387 37334 storage/stores.go:295  [n1] read 0 node addresses from persistent storage
I170726 14:59:23.669427 37334 server/node.go:586  [n1] connecting to gossip network to verify cluster ID...
I170726 14:59:23.669449 37334 server/node.go:611  [n1] node connected via gossip and verified as part of cluster "62b205ee-8db3-4ead-8cb3-6eb2bffa4361"
I170726 14:59:23.669567 37334 server/node.go:383  [n1] node=1: started with [=] engine(s) and attributes []
I170726 14:59:23.669619 37334 sql/executor.go:419  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:45878}
I170726 14:59:23.670982 37350 storage/replica_command.go:2672  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I170726 14:59:23.673802 37334 server/server.go:826  [n1] starting https server at 127.0.0.1:59767
I170726 14:59:23.673827 37334 server/server.go:827  [n1] starting grpc/postgres server at 127.0.0.1:45878
I170726 14:59:23.673841 37334 server/server.go:828  [n1] advertising CockroachDB node at 127.0.0.1:45878
E170726 14:59:23.686823 37351 storage/queue.go:658  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170726 14:59:23.687245 37350 storage/replica_command.go:2672  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/tsd [r3]
E170726 14:59:23.692606 37499 storage/replica_proposal.go:539  [n1,s1,r2/1:/{System/-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170726 14:59:23.693029 37350 storage/replica_command.go:2672  [split,n1,s1,r3/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r4]
E170726 14:59:23.699127 37476 storage/replica_proposal.go:539  [n1,s1,r3/1:/{System/tsd-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170726 14:59:23.699711 37350 storage/replica_command.go:2672  [split,n1,s1,r4/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r5]
E170726 14:59:23.710667 37501 storage/replica_proposal.go:539  [n1,s1,r4/1:/{System/tse-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170726 14:59:23.712507 37350 storage/replica_command.go:2672  [split,n1,s1,r5/1:/{Table/System���-Max}] initiating a split of this range at key /Table/11 [r6]
I170726 14:59:23.715132 37334 sql/event_log.go:101  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170726 14:59:23.717753 37334 sql/lease.go:367  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-07-26 14:59:23.717737069 +0000 UTC
I170726 14:59:23.725822 37350 storage/replica_command.go:2672  [split,n1,s1,r6/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r7]
I170726 14:59:23.732556 37334 server/server.go:961  [n1] done ensuring all necessary migrations have run
I170726 14:59:23.732580 37334 server/server.go:963  [n1] serving sql connections
I170726 14:59:23.733459 37350 storage/replica_command.go:2672  [split,n1,s1,r7/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r8]
I170726 14:59:23.740660 37770 sql/event_log.go:101  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45878} Attrs: Locality:} ClusterID:62b205ee-8db3-4ead-8cb3-6eb2bffa4361 StartedAt:1501081163669549474 LastUp:1501081163669549474}
I170726 14:59:23.748923 37350 storage/replica_command.go:2672  [split,n1,s1,r8/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r9]
I170726 14:59:23.766381 37350 storage/replica_command.go:2672  [split,n1,s1,r9/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r10]
I170726 14:59:23.788257 37935 sql/event_log.go:101  [client=127.0.0.1:51946,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:data Statement:CREATE DATABASE IF NOT EXISTS data User:root}
I170726 14:59:23.789307 37350 storage/replica_command.go:2672  [split,n1,s1,r10/1:/{Table/15-Max}] initiating a split of this range at key /Table/50 [r11]
I170726 14:59:23.793561 37935 sql/event_log.go:101  [client=127.0.0.1:51946,user=root,n1] Event: "create_table", target: 51, info: {TableName:data.bank Statement:CREATE TABLE data.bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I170726 14:59:23.793854 37350 storage/replica_command.go:2672  [split,n1,s1,r11/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r12]
I170726 14:59:23.797781 37871 storage/replica_command.go:2672  [n1,s1,r11/1:/{Table/50-Max}] initiating a split of this range at key /Table/51/1/0 [r13]
I170726 14:59:23.848434 38055 storage/replica_command.go:2672  [n1,s1,r12/1:/{Table/51-Max}] initiating a split of this range at key /Table/51/1/0 [r14]
I170726 14:59:23.853146 38069 storage/replica_command.go:2672  [n1,s1,r14/1:/{Table/51/1/0-Max}] initiating a split of this range at key /Table/51/1/1 [r15]
I170726 14:59:23.859058 38102 storage/replica_command.go:2672  [n1,s1,r15/1:/{Table/51/1/1-Max}] initiating a split of this range at key /Table/51/1/2 [r16]
I170726 14:59:23.870484 37911 storage/replica_command.go:2672  [n1,s1,r16/1:/{Table/51/1/2-Max}] initiating a split of this range at key /Table/51/1/3 [r17]
I170726 14:59:23.876176 38136 storage/replica_command.go:2672  [n1,s1,r17/1:/{Table/51/1/3-Max}] initiating a split of this range at key /Table/51/1/4 [r18]
I170726 14:59:23.880665 38142 storage/replica_command.go:2672  [n1,s1,r18/1:/{Table/51/1/4-Max}] initiating a split of this range at key /Table/51/1/5 [r19]
I170726 14:59:23.893801 38201 storage/replica_command.go:2672  [n1,s1,r19/1:/{Table/51/1/5-Max}] initiating a split of this range at key /Table/51/1/6 [r20]
I170726 14:59:23.908116 38152 storage/replica_command.go:2672  [n1,s1,r20/1:/{Table/51/1/6-Max}] initiating a split of this range at key /Table/51/1/7 [r21]
I170726 14:59:23.912976 38094 storage/replica_command.go:2672  [n1,s1,r21/1:/{Table/51/1/7-Max}] initiating a split of this range at key /Table/51/1/8 [r22]
I170726 14:59:23.924201 37935 sql/event_log.go:101  [client=127.0.0.1:51946,user=root,n1] Event: "create_table", target: 52, info: {TableName:i0 Statement:CREATE TABLE i0 (a INT, b INT, PRIMARY KEY (a, b)) INTERLEAVE IN PARENT bank (a) User:root}
I170726 14:59:23.926492 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=52 (i0) version=2 mtime=2017-07-26 14:59:23.926473349 +0000 UTC
I170726 14:59:23.928322 37350 storage/replica_command.go:2672  [split,n1,s1,r22/1:/{Table/51/1/8-Max}] initiating a split of this range at key /Table/52 [r23]
I170726 14:59:23.933286 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=51 (bank) version=2 mtime=2017-07-26 14:59:23.93326879 +0000 UTC
I170726 14:59:23.935214 37935 sql/lease.go:296  publish (count leases): descID=51 name=bank version=1 count=1
I170726 14:59:23.974515 37935 sql/event_log.go:101  [client=127.0.0.1:51946,user=root,n1] Event: "create_table", target: 53, info: {TableName:i0_0 Statement:CREATE TABLE i0_0 (a INT, b INT, c INT, PRIMARY KEY (a, b, c)) INTERLEAVE IN PARENT i0 (a, b) User:root}
I170726 14:59:23.976457 37350 storage/replica_command.go:2672  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
I170726 14:59:23.977200 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=53 (i0_0) version=2 mtime=2017-07-26 14:59:23.97718126 +0000 UTC
I170726 14:59:23.983982 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=52 (i0) version=3 mtime=2017-07-26 14:59:23.983962301 +0000 UTC
I170726 14:59:24.028798 37935 sql/event_log.go:101  [client=127.0.0.1:51946,user=root,n1] Event: "create_table", target: 54, info: {TableName:i1 Statement:CREATE TABLE i1 (a INT, b INT, PRIMARY KEY (a, b)) INTERLEAVE IN PARENT bank (a) User:root}
I170726 14:59:24.029720 37350 storage/replica_command.go:2672  [split,n1,s1,r24/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r25]
I170726 14:59:24.033456 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=54 (i1) version=2 mtime=2017-07-26 14:59:24.033437106 +0000 UTC
I170726 14:59:24.042511 37935 sql/lease.go:367  [client=127.0.0.1:51946,user=root,n1] publish: descID=51 (bank) version=3 mtime=2017-07-26 14:59:24.042493361 +0000 UTC
I170726 14:59:24.044434 37935 sql/lease.go:296  publish (count leases): descID=51 name=bank version=2 count=1
I170726 14:59:24.124490 38809 storage/replica_command.go:2672  [n1,s1,r14/1:/Table/51/1/{0-1}] initiating a split of this range at key /Table/51/1/0/#/52/1/1 [r26]
I170726 14:59:24.130094 38842 storage/replica_command.go:2672  [n1,s1,r26/1:/Table/51/1/{0/#/52���-1}] initiating a split of this range at key /Table/51/1/0/#/52/1/2 [r27]
I170726 14:59:24.134790 38592 storage/replica_command.go:2672  [n1,s1,r15/1:/Table/51/1/{1-2}] initiating a split of this range at key /Table/51/1/1/#/52/1/1 [r28]
I170726 14:59:24.142509 38727 storage/replica_command.go:2672  [n1,s1,r28/1:/Table/51/1/{1/#/52���-2}] initiating a split of this range at key /Table/51/1/1/#/52/1/2 [r29]
I170726 14:59:24.148629 38736 storage/replica_command.go:2672  [n1,s1,r16/1:/Table/51/1/{2-3}] initiating a split of this range at key /Table/51/1/2/#/52/1/1 [r30]
I170726 14:59:24.164037 38925 storage/replica_command.go:2672  [n1,s1,r30/1:/Table/51/1/{2/#/52���-3}] initiating a split of this range at key /Table/51/1/2/#/52/1/2 [r31]
I170726 14:59:24.168789 38892 storage/replica_command.go:2672  [n1,s1,r17/1:/Table/51/1/{3-4}] initiating a split of this range at key /Table/51/1/3/#/52/1/1 [r32]
I170726 14:59:24.185624 38875 storage/replica_command.go:2672  [n1,s1,r32/1:/Table/51/1/{3/#/52���-4}] initiating a split of this range at key /Table/51/1/3/#/52/1/2 [r33]
I170726 14:59:24.192040 38956 storage/replica_command.go:2672  [n1,s1,r18/1:/Table/51/1/{4-5}] initiating a split of this range at key /Table/51/1/4/#/52/1/1 [r34]
I170726 14:59:24.198775 38971 storage/replica_command.go:2672  [n1,s1,r34/1:/Table/51/1/{4/#/52���-5}] initiating a split of this range at key /Table/51/1/4/#/52/1/2 [r35]
I170726 14:59:24.211073 39064 storage/replica_command.go:2672  [n1,s1,r26/1:/Table/51/1/0/#/52/1/{1-2}] initiating a split of this range at key /Table/51/1/0/#/52/1/1/#/53/1 [r36]
I170726 14:59:24.230859 39143 storage/replica_command.go:2672  [n1,s1,r27/1:/Table/51/1/{0/#/52���-1}] initiating a split of this range at key /Table/51/1/0/#/52/1/2/#/53/1 [r37]
I170726 14:59:24.236754 39051 storage/replica_command.go:2672  [n1,s1,r28/1:/Table/51/1/1/#/52/1/{1-2}] initiating a split of this range at key /Table/51/1/1/#/52/1/1/#/53/1 [r38]
I170726 14:59:24.244469 39094 storage/replica_command.go:2672  [n1,s1,r29/1:/Table/51/1/{1/#/52���-2}] initiating a split of this range at key /Table/51/1/1/#/52/1/2/#/53/1 [r39]
I170726 14:59:24.258055 39163 storage/replica_command.go:2672  [n1,s1,r30/1:/Table/51/1/2/#/52/1/{1-2}] initiating a split of this range at key /Table/51/1/2/#/52/1/1/#/53/1 [r40]
I170726 14:59:24.265432 39205 storage/replica_command.go:2672  [n1,s1,r31/1:/Table/51/1/{2/#/52���-3}] initiating a split of this range at key /Table/51/1/2/#/52/1/2/#/53/1 [r41]
I170726 14:59:24.275403 39226 storage/replica_command.go:2672  [n1,s1,r32/1:/Table/51/1/3/#/52/1/{1-2}] initiating a split of this range at key /Table/51/1/3/#/52/1/1/#/53/1 [r42]
I170726 14:59:24.283640 39199 storage/replica_command.go:2672  [n1,s1,r33/1:/Table/51/1/{3/#/52���-4}] initiating a split of this range at key /Table/51/1/3/#/52/1/2/#/53/1 [r43]
I170726 14:59:24.293633 39243 storage/replica_command.go:2672  [n1,s1,r34/1:/Table/51/1/4/#/52/1/{1-2}] initiating a split of this range at key /Table/51/1/4/#/52/1/1/#/53/1 [r44]
I170726 14:59:24.298327 39259 storage/replica_command.go:2672  [n1,s1,r35/1:/Table/51/1/{4/#/52���-5}] initiating a split of this range at key /Table/51/1/4/#/52/1/2/#/53/1 [r45]
I170726 14:59:24.310670 39341 storage/replica_command.go:2672  [n1,s1,r37/1:/Table/51/1/{0/#/52���-1}] initiating a split of this range at key /Table/51/1/0/#/54/1/1 [r46]
I170726 14:59:24.319495 39400 storage/replica_command.go:2672  [n1,s1,r46/1:/Table/51/1/{0/#/54���-1}] initiating a split of this range at key /Table/51/1/0/#/54/1/2 [r47]
I170726 14:59:24.324196 39424 storage/replica_command.go:2672  [n1,s1,r39/1:/Table/51/1/{1/#/52���-2}] initiating a split of this range at key /Table/51/1/1/#/54/1/1 [r48]
I170726 14:59:24.336232 39405 storage/replica_command.go:2672  [n1,s1,r48/1:/Table/51/1/{1/#/54���-2}] initiating a split of this range at key /Table/51/1/1/#/54/1/2 [r49]
I170726 14:59:24.361879 39111 storage/replica_command.go:2672  [n1,s1,r41/1:/Table/51/1/{2/#/52���-3}] initiating a split of this range at key /Table/51/1/2/#/54/1/1 [r50]
I170726 14:59:24.366460 39483 storage/replica_command.go:2672  [n1,s1,r50/1:/Table/51/1/{2/#/54���-3}] initiating a split of this range at key /Table/51/1/2/#/54/1/2 [r51]
I170726 14:59:24.389589 39494 storage/replica_command.go:2672  [n1,s1,r43/1:/Table/51/1/{3/#/52���-4}] initiating a split of this range at key /Table/51/1/3/#/54/1/1 [r52]
I170726 14:59:24.397451 39529 storage/replica_command.go:2672  [n1,s1,r52/1:/Table/51/1/{3/#/54���-4}] initiating a split of this range at key /Table/51/1/3/#/54/1/2 [r53]
I170726 14:59:24.406908 39555 storage/replica_command.go:2672  [n1,s1,r45/1:/Table/51/1/{4/#/52���-5}] initiating a split of this range at key /Table/51/1/4/#/54/1/1 [r54]
I170726 14:59:24.414295 39441 storage/replica_command.go:2672  [n1,s1,r54/1:/Table/51/1/{4/#/54���-5}] initiating a split of this range at key /Table/51/1/4/#/54/1/2 [r55]
I170726 14:59:24.425580 39630 ccl/storageccl/export.go:129  [n1,s1,r26/1:/Table/51/1/0/#/52/1/1{-/���}] export [/Table/51/1/0/#/52/1/1,/Table/51/1/0/#/52/1/1/#/53/1)
I170726 14:59:24.425722 39518 ccl/storageccl/export.go:129  [n1,s1,r12/1:/Table/51{-/1/0}] export [/Table/51/1,/Table/51/1/0)
I170726 14:59:24.425753 39473 ccl/storageccl/export.go:129  [n1,s1,r5/1:/Table/{SystemCon���-11}] export [/Table/3/1,/Table/3/2)
I170726 14:59:24.426023 39632 ccl/storageccl/export.go:129  [n1,s1,r36/1:/Table/51/1/0/#/52/1/{1/���-2}] export [/Table/51/1/0/#/52/1/1/#/53/1,/Table/51/1/0/#/52/1/2)
I170726 14:59:24.426220 39635 ccl/storageccl/export.go:129  [n1,s1,r27/1:/Table/51/1/0/#/52/1/2{-/���}] export [/Table/51/1/0/#/52/1/2,/Table/51/1/0/#/52/1/2/#/53/1)
I170726 14:59:24.426581 39650 ccl/storageccl/export.go:129  [n1,s1,r14/1:/Table/51/1/0{-/#/52/���}] export [/Table/51/1/0,/Table/51/1/0/#/52/1/1)
I170726 14:59:24.426615 39637 ccl/storageccl/export.go:129  [n1,s1,r37/1:/Table/51/1/0/#/5{2/1/���-4/1/1}] export [/Table/51/1/0/#/52/1/2/#/53/1,/Table/51/1/0/#/54/1/1)
I170726 14:59:24.426890 39652 ccl/storageccl/export.go:129  [n1,s1,r46/1:/Table/51/1/0/#/54/1/{1-2}] export [/Table/51/1/0/#/54/1/1,/Table/51/1/0/#/54/1/2)
I170726 14:59:24.427419 39655 ccl/storageccl/export.go:129  [n1,s1,r47/1:/Table/51/1/{0/#/54���-1}] export [/Table/51/1/0/#/54/1/2,/Table/51/1/1)
I170726 14:59:24.427713 39657 ccl/storageccl/export.go:129  [n1,s1,r15/1:/Table/51/1/1{-/#/52/���}] export [/Table/51/1/1,/Table/51/1/1/#/52/1/1)
I170726 14:59:24.429728 39489 ccl/storageccl/export.go:129  [n1,s1,r5/1:/Table/{SystemCon���-11}] export [/Table/4/1,/Table/4/2)
I170726 14:59:24.429927 39699 ccl/storageccl/export.go:129  [n1,s1,r28/1:/Table/51/1/1/#/52/1/1{-/���}] export [/Table/51/1/1/#/52/1/1,/Table/51/1/1/#/52/1/1/#/53/1)
I170726 14:59:24.430297 39701 ccl/storageccl/export.go:129  [n1,s1,r38/1:/Table/51/1/1/#/52/1/{1/���-2}] export [/Table/51/1/1/#/52/1/1/#/53/1,/Table/51/1/1/#/52/1/2)
I170726 14:59:24.443825 39799 ccl/storageccl/export.go:129  [n1,s1,r29/1:/Table/51/1/1/#/52/1/2{-/���}] export [/Table/51/1/1/#/52/1/2,/Table/51/1/1/#/52/1/2/#/53/1)
I170726 14:59:24.444169 39801 ccl/storageccl/export.go:129  [n1,s1,r39/1:/Table/51/1/1/#/5{2/1/���-4/1/1}] export [/Table/51/1/1/#/52/1/2/#/53/1,/Table/51/1/1/#/54/1/1)
I170726 14:59:24.449802 39803 ccl/storageccl/export.go:129  [n1,s1,r48/1:/Table/51/1/1/#/54/1/{1-2}] export [/Table/51/1/1/#/54/1/1,/Table/51/1/1/#/54/1/2)
I170726 14:59:24.461208 39769 ccl/storageccl/export.go:129  [n1,s1,r49/1:/Table/51/1/{1/#/54���-2}] export [/Table/51/1/1/#/54/1/2,/Table/51/1/2)
I170726 14:59:24.463086 39845 ccl/storageccl/export.go:129  [n1,s1,r16/1:/Table/51/1/2{-/#/52/���}] export [/Table/51/1/2,/Table/51/1/2/#/52/1/1)
I170726 14:59:24.463491 39847 ccl/storageccl/export.go:129  [n1,s1,r30/1:/Table/51/1/2/#/52/1/1{-/���}] export [/Table/51/1/2/#/52/1/1,/Table/51/1/2/#/52/1/1/#/53/1)
I170726 14:59:24.468638 39823 ccl/storageccl/export.go:129  [n1,s1,r40/1:/Table/51/1/2/#/52/1/{1/���-2}] export [/Table/51/1/2/#/52/1/1/#/53/1,/Table/51/1/2/#/52/1/2)
I170726 14:59:24.468976 39825 ccl/storageccl/export.go:129  [n1,s1,r31/1:/Table/51/1/2/#/52/1/2{-/���}] export [/Table/51/1/2/#/52/1/2,/Table/51/1/2/#/52/1/2/#/53/1)
I170726 14:59:24.469449 39893 ccl/storageccl/export.go:129  [n1,s1,r41/1:/Table/51/1/2/#/5{2/1/���-4/1/1}] export [/Table/51/1/2/#/52/1/2/#/53/1,/Table/51/1/2/#/54/1/1)
I170726 14:59:24.482995 39882 ccl/storageccl/export.go:129  [n1,s1,r50/1:/Table/51/1/2/#/54/1/{1-2}] export [/Table/51/1/2/#/54/1/1,/Table/51/1/2/#/54/1/2)
I170726 14:59:24.483535 39884 ccl/storageccl/export.go:129  [n1,s1,r51/1:/Table/51/1/{2/#/54���-3}] export [/Table/51/1/2/#/54/1/2,/Table/51/1/3)
I170726 14:59:24.483869 39886 ccl/storageccl/export.go:129  [n1,s1,r17/1:/Table/51/1/3{-/#/52/���}] export [/Table/51/1/3,/Table/51/1/3/#/52/1/1)
I170726 14:59:24.489961 39928 ccl/storageccl/export.go:129  [n1,s1,r32/1:/Table/51/1/3/#/52/1/1{-/���}] export [/Table/51/1/3/#/52/1/1,/Table/51/1/3/#/52/1/1/#/53/1)
I170726 14:59:24.492881 39930 ccl/storageccl/export.go:129  [n1,s1,r42/1:/Table/51/1/3/#/52/1/{1/���-2}] export [/Table/51/1/3/#/52/1/1/#/53/1,/Table/51/1/3/#/52/1/2)
I170726 14:59:24.493489 39932 ccl/storageccl/export.go:129  [n1,s1,r33/1:/Table/51/1/3/#/52/1/2{-/���}] export [/Table/51/1/3/#/52/1/2,/Table/51/1/3/#/52/1/2/#/53/1)
I170726 14:59:24.495413 37292 ccl/storageccl/export.go:129  [n1,s1,r43/1:/Table/51/1/3/#/5{2/1/���-4/1/1}] export [/Table/51/1/3/#/52/1/2/#/53/1,/Table/51/1/3/#/54/1/1)
I170726 14:59:24.495978 37296 ccl/storageccl/export.go:129  [n1,s1,r52/1:/Table/51/1/3/#/54/1/{1-2}] export [/Table/51/1/3/#/54/1/1,/Table/51/1/3/#/54/1/2)
I170726 14:59:24.496317 40034 ccl/storageccl/export.go:129  [n1,s1,r53/1:/Table/51/1/{3/#/54���-4}] export [/Table/51/1/3/#/54/1/2,/Table/51/1/4)
I170726 14:59:24.506431 39953 ccl/storageccl/export.go:129  [n1,s1,r18/1:/Table/51/1/4{-/#/52/���}] export [/Table/51/1/4,/Table/51/1/4/#/52/1/1)
I170726 14:59:24.511155 40115 ccl/storageccl/export.go:129  [n1,s1,r34/1:/Table/51/1/4/#/52/1/1{-/���}] export [/Table/51/1/4/#/52/1/1,/Table/51/1/4/#/52/1/1/#/53/1)
I170726 14:59:24.511523 40117 ccl/storageccl/export.go:129  [n1,s1,r44/1:/Table/51/1/4/#/52/1/{1/���-2}] export [/Table/51/1/4/#/52/1/1/#/53/1,/Table/51/1/4/#/52/1/2)
I170726 14:59:24.517337 40163 ccl/storageccl/export.go:129  [n1,s1,r35/1:/Table/51/1/4/#/52/1/2{-/���}] export [/Table/51/1/4/#/52/1/2,/Table/51/1/4/#/52/1/2/#/53/1)
I170726 14:59:24.517933 40165 ccl/storageccl/export.go:129  [n1,s1,r45/1:/Table/51/1/4/#/5{2/1/���-4/1/1}] export [/Table/51/1/4/#/52/1/2/#/53/1,/Table/51/1/4/#/54/1/1)
I170726 14:59:24.518279 40167 ccl/storageccl/export.go:129  [n1,s1,r54/1:/Table/51/1/4/#/54/1/{1-2}] export [/Table/51/1/4/#/54/1/1,/Table/51/1/4/#/54/1/2)
I170726 14:59:24.525315 40229 ccl/storageccl/export.go:129  [n1,s1,r55/1:/Table/51/1/{4/#/54���-5}] export [/Table/51/1/4/#/54/1/2,/Table/51/1/5)
I170726 14:59:24.526014 40231 ccl/storageccl/export.go:129  [n1,s1,r19/1:/Table/51/1/{5-6}] export [/Table/51/1/5,/Table/51/1/6)
I170726 14:59:24.545153 39741 ccl/storageccl/export.go:129  [n1,s1,r20/1:/Table/51/1/{6-7}] export [/Table/51/1/6,/Table/51/1/7)
I170726 14:59:24.555527 40295 ccl/storageccl/export.go:129  [n1,s1,r21/1:/Table/51/1/{7-8}] export [/Table/51/1/7,/Table/51/1/8)
I170726 14:59:24.555910 40297 ccl/storageccl/export.go:129  [n1,s1,r22/1:/Table/5{1/1/8-2}] export [/Table/51/1/8,/Table/51/2)
--- FAIL: test/TestBackupRestoreInterleaved/all_tables_in_interleave_hierarchy (0.710s)
backup_test.go:700: expected 10 rows but found 9
backup_test.go:708: expected 30 rows but found 29
------- Stdout: -------
W170726 14:59:24.596204 40441 server/server.go:303  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170726 14:59:24.596393 40441 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170726 14:59:24.602531 40441 server/config.go:523  [n?] 1 storage engine initialized
I170726 14:59:24.602553 40441 server/config.go:525  [n?] RocksDB cache size: 512 MiB
I170726 14:59:24.602563 40441 server/config.go:525  [n?] store 0: in-memory, size 100 MiB
I170726 14:59:24.602793 40441 server/node.go:432  [n?] store [n0,s0] not bootstrapped
I170726 14:59:24.604712 40441 server/node.go:367  [n?] **** cluster 7e7de985-4126-4532-8da7-110f31a11d8f has been created
I170726 14:59:24.604731 40441 server/node.go:368  [n?] **** add additional nodes by specifying --join=127.0.0.1:52752
I170726 14:59:24.605129 40441 storage/store.go:1262  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170726 14:59:24.605158 40441 server/node.go:445  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0 WritesPerSecond:0}
I170726 14:59:24.605183 40441 server/node.go:329  [n1] node ID 1 initialized
I170726 14:59:24.605220 40441 gossip/gossip.go:297  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:52752" > attrs:<> locality:<> 
I170726 14:59:24.605363 40441 storage/stores.go:295  [n1] read 0 node addresses from persistent storage
I170726 14:59:24.605402 40441 server/node.go:586  [n1] connecting to gossip network to verify cluster ID...
I170726 14:59:24.605699 40441 server/node.go:611  [n1] node connected via gossip and verified as part of cluster "7e7de985-4126-4532-8da7-110f31a11d8f"
I170726 14:59:24.605725 40441 server/node.go:383  [n1] node=1: started with [=] engine(s) and attributes []
I170726 14:59:24.605775 40441 sql/executor.go:419  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:52752}
I170726 14:59:24.608470 40441 server/server.go:826  [n1] starting https server at 127.0.0.1:60065
I170726 14:59:24.608492 40441 server/server.go:827  [n1] starting grpc/postgres server at 127.0.0.1:52752
I170726 14:59:24.608500 40441 server/server.go:828  [n1] advertising CockroachDB node at 127.0.0.1:52752
I170726 14:59:24.622030 40408 storage/replica_command.go:2672  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I170726 14:59:24.630799 40408 storage/replica_command.go:2672  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/tsd [r3]
E170726 14:59:24.631217 40409 storage/queue.go:658  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
E170726 14:59:24.640515 40409 storage/queue.go:658  [replicate,n1,s1,r2/1:/System/{-tsd}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170726 14:59:24.640615 40408 storage/replica_command.go:2672  [split,n1,s1,r3/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r4]
E170726 14:59:24.645033 40409 storage/queue.go:658  [replicate,n1,s1,r3/1:/System/ts{d-e}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170726 14:59:24.645169 40408 storage/replica_command.go:2672  [split,n1,s1,r4/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r5]
E170726 14:59:24.649052 40409 storage/queue.go:658  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I170726 14:59:24.653307 40408 storage/replica_command.go:2672  [split,n1,s1,r5/1:/{Table/System���-Max}] initiating a split of this range at key /Table/11 [r6]
I170726 14:59:24.657528 40408 storage/replica_command.go:2672  [split,n1,s1,r6/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r7]
I170726 14:59:24.664323 40408 storage/replica_command.go:2672  [split,n1,s1,r7/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r8]
I170726 14:59:24.670447 40408 storage/replica_command.go:2672  [split,n1,s1,r8/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r9]
I170726 14:59:24.681380 40441 sql/event_log.go:101  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170726 14:59:24.689731 40441 sql/lease.go:367  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-07-26 14:59:24.689712452 +0000 UTC
I170726 14:59:24.695508 40408 storage/replica_command.go:2672  [split,n1,s1,r9/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r10]
I170726 14:59:24.701452 40441 server/server.go:961  [n1] done ensuring all necessary migrations have run
I170726 14:59:24.701476 40441 server/server.go:963  [n1] serving sql connections
I170726 14:59:24.704054 40833 sql/event_log.go:101  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:52752} Attrs: Locality:} ClusterID:7e7de985-4126-4532-8da7-110f31a11d8f StartedAt:1501081164605711334 LastUp:1501081164605711334}
I170726 14:59:24.729964 40967 sql/event_log.go:101  [client=127.0.0.1:48312,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:ignored Statement:CREATE DATABASE ignored User:root}
I170726 14:59:24.730945 40408 storage/replica_command.go:2672  [split,n1,s1,r10/1:/{Table/15-Max}] initiating a split of this range at key /Table/50 [r11]
I170726 14:59:24.733823 40967 sql/event_log.go:101  [client=127.0.0.1:48312,user=root,n1] Event: "create_database", target: 51, info: {DatabaseName:data Statement:CREATE DATABASE data User:root}
I170726 14:59:24.739735 40408 storage/replica_command.go:2672  [split,n1,s1,r11/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r12]
I170726 14:59:24.750668 41095 storage/replica_command.go:2672  [n1,s1,r12/1:/{Table/51-Max}] initiating a split of this range at key /Table/52/1 [r13]
I170726 14:59:24.759634 41144 storage/replica_command.go:2672  [n1,s1,r13/1:/{Table/52/1-Max}] initiating a split of this range at key /Table/52/1/0 [r14]
I170726 14:59:24.769556 41181 storage/replica_command.go:2672  [n1,s1,r14/1:/{Table/52/1/0-Max}] initiating a split of this range at key /Table/52/1/0/#/53/1/1 [r15]
I170726 14:59:24.791373 41224 storage/replica_command.go:2672  [n1,s1,r15/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/0/#/53/1/1/#/54/1 [r16]
I170726 14:59:24.798215 39969 storage/replica_command.go:2672  [n1,s1,r16/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/0/#/53/1/2 [r17]
I170726 14:59:24.803747 41271 storage/replica_command.go:2672  [n1,s1,r17/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/0/#/53/1/2/#/54/1 [r18]
I170726 14:59:24.810374 41280 storage/replica_command.go:2672  [n1,s1,r18/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/0/#/55/1/1 [r19]
I170726 14:59:24.817678 41382 storage/replica_command.go:2672  [n1,s1,r19/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/0/#/55/1/2 [r20]
I170726 14:59:24.823404 41406 storage/replica_command.go:2672  [n1,s1,r20/1:/{Table/52/1/0���-Max}] initiating a split of this range at key /Table/52/1/1 [r21]
I170726 14:59:24.828304 41345 storage/replica_command.go:2672  [n1,s1,r21/1:/{Table/52/1/1-Max}] initiating a split of this range at key /Table/52/1/1/#/53/1/1 [r22]
I170726 14:59:24.833093 41442 storage/replica_command.go:2672  [n1,s1,r22/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/1/#/53/1/1/#/54/1 [r23]
I170726 14:59:24.846712 41424 storage/replica_command.go:2672  [n1,s1,r23/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/1/#/53/1/2 [r24]
I170726 14:59:24.853841 41519 storage/replica_command.go:2672  [n1,s1,r24/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/1/#/53/1/2/#/54/1 [r25]
I170726 14:59:24.860556 41438 storage/replica_command.go:2672  [n1,s1,r25/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/1/#/55/1/1 [r26]
I170726 14:59:24.879656 41619 storage/replica_command.go:2672  [n1,s1,r26/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/1/#/55/1/2 [r27]
I170726 14:59:24.888167 41661 storage/replica_command.go:2672  [n1,s1,r27/1:/{Table/52/1/1���-Max}] initiating a split of this range at key /Table/52/1/2 [r28]
I170726 14:59:24.903319 41730 storage/replica_command.go:2672  [n1,s1,r28/1:/{Table/52/1/2-Max}] initiating a split of this range at key /Table/52/1/2/#/53/1/1 [r29]
I170726 14:59:24.910931 41641 storage/replica_command.go:2672  [n1,s1,r29/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/2/#/53/1/1/#/54/1 [r30]
I170726 14:59:24.937510 41804 storage/replica_command.go:2672  [n1,s1,r30/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/2/#/53/1/2 [r31]
I170726 14:59:24.945148 41758 storage/replica_command.go:2672  [n1,s1,r31/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/2/#/53/1/2/#/54/1 [r32]
I170726 14:59:24.953311 41850 storage/replica_command.go:2672  [n1,s1,r32/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/2/#/55/1/1 [r33]
I170726 14:59:24.959544 41882 storage/replica_command.go:2672  [n1,s1,r33/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/2/#/55/1/2 [r34]
I170726 14:59:24.977171 41901 storage/replica_command.go:2672  [n1,s1,r34/1:/{Table/52/1/2���-Max}] initiating a split of this range at key /Table/52/1/3 [r35]
I170726 14:59:24.991863 41918 storage/replica_command.go:2672  [n1,s1,r35/1:/{Table/52/1/3-Max}] initiating a split of this range at key /Table/52/1/3/#/53/1/1 [r36]
I170726 14:59:25.000028 41936 storage/replica_command.go:2672  [n1,s1,r36/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/3/#/53/1/1/#/54/1 [r37]
I170726 14:59:25.012106 42018 storage/replica_command.go:2672  [n1,s1,r37/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/3/#/53/1/2 [r38]
I170726 14:59:25.017818 42051 storage/replica_command.go:2672  [n1,s1,r38/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/3/#/53/1/2/#/54/1 [r39]
I170726 14:59:25.024878 42086 storage/replica_command.go:2672  [n1,s1,r39/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/3/#/55/1/1 [r40]
I170726 14:59:25.031744 42133 storage/replica_command.go:2672  [n1,s1,r40/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/3/#/55/1/2 [r41]
I170726 14:59:25.042182 42173 storage/replica_command.go:2672  [n1,s1,r41/1:/{Table/52/1/3���-Max}] initiating a split of this range at key /Table/52/1/4 [r42]
I170726 14:59:25.061690 42203 storage/replica_command.go:2672  [n1,s1,r42/1:/{Table/52/1/4-Max}] initiating a split of this range at key /Table/52/1/4/#/53/1/1 [r43]
I170726 14:59:25.068191 42248 storage/replica_command.go:2672  [n1,s1,r43/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/4/#/53/1/1/#/54/1 [r44]
I170726 14:59:25.076696 41831 storage/replica_command.go:2672  [n1,s1,r44/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/4/#/53/1/2 [r45]
I170726 14:59:25.082154 42343 storage/replica_command.go:2672  [n1,s1,r45/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/4/#/53/1/2/#/54/1 [r46]
I170726 14:59:25.089404 42237 storage/replica_command.go:2672  [n1,s1,r46/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/4/#/55/1/1 [r47]
I170726 14:59:25.103860 42421 storage/replica_command.go:2672  [n1,s1,r47/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/4/#/55/1/2 [r48]
I170726 14:59:25.113645 42361 storage/replica_command.go:2672  [n1,s1,r48/1:/{Table/52/1/4���-Max}] initiating a split of this range at key /Table/52/1/5 [r49]
I170726 14:59:25.136400 42445 storage/replica_command.go:2672  [n1,s1,r49/1:/{Table/52/1/5-Max}] initiating a split of this range at key /Table/52/1/6 [r50]
I170726 14:59:25.153607 42515 storage/replica_command.go:2672  [n1,s1,r50/1:/{Table/52/1/6-Max}] initiating a split of this range at key /Table/52/1/7 [r51]
I170726 14:59:25.170059 42065 storage/replica_command.go:2672  [n1,s1,r51/1:/{Table/52/1/7-Max}] initiating a split of this range at key /Table/52/1/8 [r52]
I170726 14:59:25.293527 40235 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:52752->127.0.0.1:48306: use of closed network connection

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/ccl/sqlccl/restore.go
+++ b/pkg/ccl/sqlccl/restore.go
@@ -708,7 +708,10 @@ func restore(
 	})
 
 	var importIdx int
-	for importRequest := range importRequestsCh {
+	for ir := range importRequestsCh {
+		// Copy ir so we can use it in the goroutine below.
+		importRequest := ir
+
 		importCtx, importSpan := tracing.ChildSpan(gCtx, "import")
 		idx := importIdx
 		importIdx++

--- a/pkg/ccl/storageccl/import.go
+++ b/pkg/ccl/storageccl/import.go
@@ -159,7 +159,7 @@ func (b *sstBatcher) Finish(ctx context.Context, db *client.DB) error {
 
 	const maxAddSSTableRetries = 10
 	for i := 0; ; i++ {
-		log.Event(ctx, "sending AddSSTable")
+		log.VEventf(ctx, 2, "sending AddSSTable [%s,%s)", start, end)
 		// TODO(dan): This will fail if the range has split.
 		err := db.ExperimentalAddSSTable(ctx, start, end, sstBytes)
 		if err == nil {
@@ -208,7 +208,7 @@ func evalImport(ctx context.Context, cArgs storage.CommandArgs) (*roachpb.Import
 	var rows rowCounter
 	var iters []engine.SimpleIterator
 	for _, file := range args.Files {
-		log.VEventf(ctx, 2, "import file %s", file.Path)
+		log.VEventf(ctx, 2, "import file %s %s", file.Path, args.Span.Key)
 
 		dir, err := MakeExportStorage(ctx, file.Dir)
 		if err != nil {

